pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -145,7 +145,7 @@ stdenv.mkDerivation {
       }
 
       # Install git-subtree.
-      make -C contrib/subtree install ${stdenv.lib.optionalString withManual "install-doc"}
+      make -C contrib/subtree install ${stdenv.lib.optionalString withManual "USE_ASCIIDOCTOR=1 install-doc"}
       rm -rf contrib/subtree
```

Comment:
this should be added with the existing installFlags
```
installFlags = [ "NO_INSTALL_HARDLINKS=1" "USE_ASCIIDOCTOR=1" ];
```

Reply:
I just tried that, but `installFlags` is not mentioned in the make install invocation for `contrib/subtree`, so it fails trying to fall back to asciidoc. Same thing on line 234:

```nix
   + stdenv.lib.optionalString withManual ''# Install man pages and Info manual
       make -j $NIX_BUILD_CORES -l $NIX_BUILD_CORES USE_ASCIIDOCTOR=1 PERL_PATH="${buildPackages.perl}/bin/perl" cmd-list.made install install-html install-info \
         -C Documentation ''
```

Reply:
ah, that's fair.  forgot about that